### DISCUSSION TITLE
Channel permissions

### DISCUSSION THREAD

**Discussion by Fliper05** [saleor/saleor#9857] [Deployment]
Hello there, I was wondering if it would be possible to make a channel hidden, so that only the selected users could access it. We want to use channels to differentiate between wholesale and retail, as the prices are different. Could anyone of you please guide me with some resources to achieve this?

**Comment by zakokor** [comment]
Same needs, thanks

**Comment by timuric** [comment]
It is currently in progress, here is RFC, feel free to comment https://github.com/saleor/saleor/pull/10423

**Reply by zakokor** [reply]
Thanks Timur, looks like what we need, any expectation of it being released soon?
